src/server/routes/category.js

```diff
@@ -22,15 +22,18 @@ import categories from '../../data/categories';
 
 const category = (req, res, next) => {
   const categoryId = req.params.id;
-  fbAdmin.database().ref('/products')
-    .orderByChild('category')
-    .equalTo(categoryId)
-    .once('value')
+  fbAdmin.firestore().collection('products').where('category', '==', categoryId).get()
```

Comment:
Should this be `'==='`?

Reply:
I don't think so, as it's a Firestore operator and not js:
"The operation string (e.g "<", "<=", "==", ">", ">=")
https://firebase.google.com/docs/reference/js/firebase.firestore.Query#where